<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowOptions</class>
 <widget class="QWidget" name="MainWindowOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оформление</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabCommands">
      <attribute name="title">
       <string>Команды</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,1">
       <item>
        <widget class="QGroupBox" name="groupBoxCommands">
         <property name="title">
          <string>Категория / Команды:</string>
         </property>
         <layout class="QHBoxLayout" name="layoutCommands" stretch="2,3">
          <item>
           <widget class="QListView" name="listCategory">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ListView" name="listCommands">
            <property name="styleSheet">
             <string notr="true">background: palette(Window);</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxDescription">
         <property name="title">
          <string>Описание: </string>
         </property>
         <layout class="QVBoxLayout" name="layoutDescription">
          <item>
           <widget class="QLabel" name="labelDescription">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabToolBars">
      <attribute name="title">
       <string>Панели инструментов</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" rowspan="5">
        <widget class="QListWidget" name="listToolBars"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelToolBars">
         <property name="text">
          <string>Панели инструментов:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonNew">
         <property name="text">
          <string>Новая...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonRename">
         <property name="text">
          <string>Переименовать...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutPushButton">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>listview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
